bitnami/kafka/values.yaml

```diff
@@ -942,6 +942,21 @@ metrics:
       ##   - myRegistryKeySecretName
       ##
       pullSecrets: []
+    ## Kafka exporter pods ServiceAccount
+    ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/
+    ##
+    serviceAccount:
+      ## @param metrics.kafka.serviceAccount.create Enable creation of ServiceAccount for Kafka exporter pods
+      ##
+      create: true
```

Comment:
I would recommend to change the default to `false`, as this could cause problems. Any concerns against that?

Reply:
I agree. As far as RBAC isn't enabled, it shouldn't make much a difference